src/docs/sdks/javascript/config/index.mdx

```diff
@@ -7,26 +7,26 @@ redirect_from:
 
 Sentry's JavaScript SDK includes many configuration options that are automatically set. You can configure your SDK using the options outlined in these pages:
 
-- **[Basic Options](/sdks/javascript/config/basics)**
+- **[Basic Options](basics)**
```

Comment:
**1. relative links**
Relative links like this works fine. I noticed that in some places you preferred adding a `./` prefix. Commenting in case you want to do the same in headers, would make grepping relative links easier (?!).

**2. canonical form, `/` suffix**
https://docs.sentry.io/sdks/javascript/config/basics redirects to https://docs.sentry.io/sdks/javascript/config/basics/, so it would be good if our links used the canonical form to avoid a redirect.